src/lib/components/WorldObjects.svelte

```diff
@@ -22,33 +22,23 @@
 </script>
 
 {#each frames.current as object (object.uuid)}
-	<Pose name={object.name}>
+	<Pose
+		name={object.name}
+		parent={object.referenceFrame}
+	>
```

Comment:
Request poses with respect to the parent reference frame now instead of world, helps simplify stuff, especially for @mattmacf98's project